website/static/js/tests/institutionsProjectSettings.test.js

```diff
@@ -0,0 +1,100 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+var assert = require('chai').assert;
+var $ = require('jquery');
+var m = require('mithril');
+var InstitutionProjectSettings = require('js/institutionProjectSettings');
+var bootbox = require('bootbox');
+
+var expectedTitle = 'Sample Project';
+
+    window.contextVars = $.extend(true, {}, window.contextVars, {
```

Comment:
Tests should not have to use window.contextVars, but I realize that you have to do this because of existing code (outside the scope of this PR), so this is fine for now.